Mathlib/Topology/Bases.lean

```diff
@@ -546,59 +559,59 @@ theorem IsSeparable.of_subtype (s : Set α) [SeparableSpace s] : IsSeparable s :
   simpa using isSeparable_range (continuous_subtype_val (p := (· ∈ s)))
 #align topological_space.is_separable_of_separable_space_subtype TopologicalSpace.IsSeparable.of_subtype
 
-@[deprecated] -- Since 2024/02/05
+@[deprecated IsSeparable.of_subtype] -- Since 2024/02/05
 alias isSeparable_of_separableSpace_subtype := IsSeparable.of_subtype
 
 theorem IsSeparable.of_separableSpace [h : SeparableSpace α] (s : Set α) : IsSeparable s :=
   IsSeparable.mono (isSeparable_univ_iff.2 h) (subset_univ _)
 #align topological_space.is_separable_of_separable_space TopologicalSpace.IsSeparable.of_separableSpace
 
-@[deprecated] -- Since 2024/02/05
+@[deprecated IsSeparable.of_separableSpace] -- Since 2024/02/05
 alias isSeparable_of_separableSpace := IsSeparable.of_separableSpace
 
 end TopologicalSpace
 
 open TopologicalSpace
 
+theorem isTopologicalBasis_iInf {β : Type*} {ι : Type*} {t : ι → TopologicalSpace β}
```

Comment:
Rename to `IsTopologicalBasis.iInf` to allow the new-style dot notation? Also, since you're adding a primed version, please add docstrings explaining the differences.